cmd/ui/src/views/Explore/EdgeInfo/EdgeInfoContent.test.tsx

```diff
@@ -87,4 +148,36 @@ describe('EdgeInfoContent', () => {
             screen.queryByText('An unexpected error has occurred. Please refresh the page and try again.')
         ).not.toBeInTheDocument();
     });
+    test('Selecting an edge with a Computer target node that haslaps is enabled shows correct Windows Abuse text', async () => {
+        render(<EdgeInfoContent selectedEdge={selectedEdgeHasLapsEnabled} />);
+
+        const user = userEvent.setup();
+        const windowAbuseAccordion = screen.getByTestId('windowsabuse-accordion');
```

Comment:
Preferably we would use the actual text/content that should be visible or not visible rather than test IDs but this is okay for now!